mono/utils/mono-threads-coop.c

```diff
@@ -794,3 +794,11 @@ mono_thread_get_coop_aware (void)
 
 	return result;
 }
+
+char mono_threads_is_runtime_startup_finished_hidden_dont_modify;
```

Comment:
Avoid contractions? “do_not”.

Reply:
Can't this just be a static gboolean? Why the `MONO_LLVM_INTERNAL_NO_EXTERN_C`?

Reply:
It is so the inline function can access it.

Reply:
Windows is using LTCG for release builds. Maybe time to try LTO for other platforms.
Then can stop inclining functions in headers.